components/Buttons/examples/ButtonsShapesExampleViewController.m

```diff
@@ -123,8 +119,7 @@ - (void)viewDidLoad {
 
   MDCButton *strokedButton = [self buildCustomStrokedButton];
   [strokedButton setTitle:@"Button" forState:UIControlStateNormal];
-  [MDCButtonTypographyThemer applyTypographyScheme:self.typographyScheme toButton:strokedButton];
-  [MDCButtonColorThemer applySemanticColorScheme:self.colorScheme toButton:strokedButton];
+  [strokedButton applyOutlinedThemeWithScheme:self.containerScheme];
```

Comment:
Tip: Nit: Switch `strokedButton` to a generic `MDCButton` and rename to `outlinedButton`?